### PR TITLE
SwiftMatrixSDK: Add swift refinements to MXSession event listeners.

### PR DESCRIPTION
**Pull request by aapierce0**
Signed-off-by: Avery Pierce <aapierce0@gmail.com>

### CODE REVIEW

File: MatrixSDK/Contrib/Swift/Data/MXSessionEventListener.swift [hunk 1]
```diff
@@ -0,0 +1,20 @@
+//
+//  MXSessionEventListener.swift
+//  MatrixSDK
+//
+//  Created by Avery Pierce on 7/12/17.
+//  Copyright © 2017 matrix.org. All rights reserved.
```
Comment (manuroe) [inline MatrixSDK/Contrib/Swift/Data/MXSessionEventListener.swift]: Unfortunately, matrix.org is not yet a foundation, nor a legal entity. Could you please remove this copyright line?
For consistency, would it be possible to use the same licence and file header as you did before (like https://github.com/matrix-org/matrix-ios-sdk/blob/6201d5709a634f7c0ea85679c04d12bf582dedab/MatrixSDK/SwiftMatrixSDK.h)?

File: MatrixSDK/Contrib/Swift/MXSession.swift [hunk 2]
```diff
@@ -275,4 +275,28 @@ public extension MXSession {
...
+    
+    /**
+     Register a global listener to events related to the current session.
+     
+     The listener will receive all events including all events of all rooms.
+     
+     - parameters:
+        - types: an array of event types to listen to
+        - block: the block that will be called once a new event has been handled.
+        - returns: a reference to use to unregister the listener
```
Comment (manuroe) [inline MatrixSDK/Contrib/Swift/MXSession.swift]: \"- returns" should have the same indentation as \"- parameters"

File: MatrixSDK/Contrib/Swift/Data/MXSessionEventListener.swift (hunk 1 above)
Comment (aapierce0) [inline MatrixSDK/Contrib/Swift/Data/MXSessionEventListener.swift]: Whoops! My mistake. I had fixed this in my local copy already, but I guess I forgot to push the change to Github before submitting the PR. It should be fixed now.

File: MatrixSDK/Contrib/Swift/MXSession.swift (hunk 2 above)
Comment (aapierce0) [inline MatrixSDK/Contrib/Swift/MXSession.swift]: Good catch! This is fixed in https://github.com/matrix-org/matrix-ios-sdk/pull/327/commits/17383a2b0138f71e5b4bd2c3e235afa9ab978dfb

**Pull request merged**